src/model/formulations/base/system.jl

```diff
@@ -2,12 +2,32 @@
 # Copyright (C) 2020, UChicago Argonne, LLC. All rights reserved.
 # Released under the modified BSD license. See COPYING.md for more details.
 
+"""
+    _add_system_wide_eqs!(model::JuMP.Model)::Nothing
+
+Calls `_add_net_injection_eqs!` and `add_reserve_eqs!`.
```

Comment:
Please remove. This comment is not very helpful and can easily get out of sync with the code.